src/styles/colors.tsx

```diff
@@ -15,6 +15,7 @@ enum Colors {
   // primary
   primary = '#5061E8',
   accent = '#5061E8',
+  lightPrimary = 'rgba(80, 97, 232, 0.1)',
```

Comment:
used for [this style](https://www.figma.com/design/PzsUxBv03Kg6GgzsSq5R1D/Mobile-Stack-x-Mento?node-id=2143-22502&node-type=instance&m=dev)

Reply:
This color can be removed and we can apply an opacity to the `expandedContainer` in `src/fiatExchanges/PaymentMethodSection.tsx`

Reply:
Your change with `lightPrimary` makes more sense as it has fewer side effects.